### DISCUSSION TITLE
Feature Request: Scrolling panel

### DISCUSSION THREAD

**Discussion by rg-gravis** [foxglove/community#1040] [General]
For designing layouts, I'd like a panel that can contain other panels (similar to the Tab panel) and is a scrollable container, for e.g. putting a lot of plots in a column and allowing you to scroll down through them to see at a glance if anything's unusual.

**Comment by defunctzombie** [comment]
How would you imagine sizing the panels?

**Reply by rg-gravis** [reply]
Wouldn't the current system of dragging on the boundaries to adjust them work? Each boundary could just adjust the height of the panel above it. I guess on panel creation there might need to be some kind of default aspect ratio.

**Reply by defunctzombie** [reply]
The current system of dragging on the boundaries and adjusting the adjacent panels works because the overall height is fixed. I suppose one could make it so that you have to adjust the first (top) panel of a stack, then the next. Another idea could be to set an equal fixed height for all panels in the scroll space and no height adjustment.

**Reply by rg-gravis** [reply]
I would prefer the first option, but either could work.
